test/OrchardCore.Commerce.Tests/CurrencyTests.cs

```diff
@@ -25,8 +25,8 @@ public static CurrencyTheoryData TestData
         [MemberData(nameof(TestData))]
         public void CurrenciesProperlyFormatAmounts(ICurrency currency, decimal amount, string expectedFormat)
         {
-            var result = currency.ToString(amount);
-            Assert.Equal(expectedFormat, result);
+            var result = currency.ToString(amount).Replace(" ", "").Replace("￥", "¥");
+            Assert.Equal(expectedFormat.Replace(" ", "").Replace("￥", "¥"), result);
```

Comment:
This is kind of ugly but there are differences in currency formats between Windows and Ubuntu (Linux).